logstash-core/src/main/java/org/logstash/secret/cli/SecretStoreCli.java

```diff
@@ -251,6 +262,9 @@ public void command(String primaryCommand, SecureConfig config, String... allArg
                 if (secretStoreFactory.exists(config.clone())) {
                     final SecretStore secretStore = secretStoreFactory.load(config);
                     for (String argument : commandLine.getArguments()) {
+                        if (!ConfigVariableExpander.KEY_PATTERN.matcher(argument).matches()) {
+                            throw new IllegalArgumentException(String.format("Invalid secret key name `%s` provided.", argument) + ConfigVariableExpander.KEY_PATTERN_DESCRIPTION);
```

Comment:
nit: I think I might have even done it in my prior review, but reading it here the mix of format strings and concatenation (~`String.format(template, var) + CONSTANT`) just _feels_ a bit weird to me; can we make room for it in the template?


```suggestion
                            throw new IllegalArgumentException(String.format("Invalid secret key name `%s` provided. %s", argument, ConfigVariableExpander.KEY_PATTERN_DESCRIPTION);
```


If not, we're missing a space after the period:

```suggestion
                            throw new IllegalArgumentException(String.format("Invalid secret key name `%s` provided. ", argument) + ConfigVariableExpander.KEY_PATTERN_DESCRIPTION);
```